src/components/Strawpoll.tsx

```diff
@@ -67,6 +67,12 @@ export interface StrawpollVoteData {
 export interface StrawpollProps extends RouteComponentProps<URLParameters> {
 } 
 
+export interface ModalProps {
+  open: boolean,
+  setOpen: Function,
```

Comment:
Also you'll want to use the `() => void` form of the function type instead of `Function`